munch-match/src/data/quizQuestions.js

```diff
@@ -0,0 +1,34 @@
+const quizQuestions = {
+    steps: [
```

Comment:
Is there a reason we need quizQuestions to be an object instead of an array? If not, we could go `const quizQuestions = [...]` and avoid needing to go `quizQuestions.steps` to access anything.